.gitignore

```diff
@@ -29,4 +29,5 @@ storage
 test/e2e/**/packages
 
 # we use corepack, no need to commit yarn binary
-.yarn
+.yarn/*
+!.yarn/patches
```

Comment:
this is not needed now, right?

Reply:
Nop, remainder from when we had a patch for linkedom, will revert

Reply:
is this really resolved?